Build/Build.cs

```diff
@@ -351,36 +355,110 @@ void ReportTestOutcome(params string[] globFilters)
 
     Target SpellCheck => _ => _
         .OnlyWhenDynamic(() => RunAllTargets || HasDocumentationChanges)
+        .DependsOn(InstallNodeWithNpm)
         .ProceedAfterFailure()
         .Executes(() =>
         {
-            Node($"{YarnCli} --silent install", workingDirectory: RootDirectory,
-                logger: YarnInstallLogger);
-
-            Node($"{YarnCli} --silent run cspell", workingDirectory: RootDirectory,
+            Npm($"install --silent", workingDirectory: RootDirectory);
+            Npm($"run --silent cspell", environmentVariables: NpmEnvironmentVariables,
                 logger: (_, msg) => Error(msg));
         });
 
-    Action<OutputType, string> YarnInstallLogger = (_, msg) =>
-    {
-        // See this PR here: https://github.com/fluentassertions/fluentassertions/pull/2537
-        // and several comments and references.
-        // This seems to be a bug in yarn and the package 'Yarn.MSBuild' is no longer maintained;
-        // So: ignore 'yarn install' errors when the same cache directory is defined in yarn.lock.
-        // This errors have the pattern like: 'Error: warning Pattern [packagename@version]...'
+    Target InstallNodeWithNpm => _ => _
+        .OnlyWhenDynamic(() => RunAllTargets || HasDocumentationChanges)
+        .DependsOn(InstallNodeWindows)
+        .DependsOn(InstallNodeOsx)
+        .DependsOn(InstallNodeLinux)
```

Comment:
Those three look a _lot_ alike. Let's see if we can deduplicate the logic.

Reply:
Yeah.. was a little tired yesterday to see that 😂